index.js

```diff
@@ -24,7 +24,7 @@ var scriptTagFinder = pred.AND(
 )
 
 var slashRegex = /([^/]+$)/;
-var localeRegex = /.*_(\w{2}).json/;
+var localeRegex = /.*_(.+).json$/;
```

Comment:
If you don't want to go quite this far:
http://stackoverflow.com/questions/7035825/regular-expression-for-a-language-tag-as-defined-by-bcp47
this would be a bit tighter if you want:
`/.*_([a-zA-z\-]+).json$/`
or for potentially more inclusivity:
`/.*_([a-zA-z0-9\-]+).json$/`